easybuild/easyconfigs/o/openmc/openmc-0.15.0_turn_off_submodule_check.patch

```diff
@@ -0,0 +1,48 @@
+diff --unified --recursive --text openmc-0.15.0.orig/CMakeLists.txt openmc-0.15.0.new/CMakeLists.txt
```

Comment:
Is this patch from upstream? If so please include a link (and a description in general)

If not, wouldn't it be easier to just remove the check instead of this large patch (due to aligning existing options) and passing an extra configure flag? If the patch is just for EB the flexibility isn't necessary, is it?

Reply:
I suppose you might want the flexibility so it can be left on when using git clone. But I've only run into this for EB so for now I've just simplified the patch and added a description.